### PR TITLE
week 16 project-labyrinth Elle Frankenberg Yennie Leek

### PR DESCRIPTION
**Pull request by ElleFrankenberg**
Link to my deployed project:
https://quirky-wiles-e65c52.netlify.app

My reflections on how this week's project turned out:

We hade some issues regarding the fetch, but we dit get some help and solved the problem

Things I'd like to have clarified or explained in more detail:
It was hard to understand how to implement the suggestion (fetch/post)  from the lecture in to our code project.

### CODE REVIEW

**Review by sirimatsdotter — APPROVED** [review]
Great work! You definitely fullfilled the blue requirements and one of the red ones. We saw your project on Friday when you were a bit stuck and hadn't come so far, so it's impressive that you catched up and produced a really nice game in the end, so great job!

We added some small comments, but overall it works and looks great with well structured code! 💯

File: code/src/components/GameScreen.js [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+import { continueGame } from '../reducers/games'
+
+
+const GameScreen = () => {
+  const actions = useSelector(store => store.games.actions)
+  const dispatch = useDispatch()
+
+  const onButtonclick = (item) => {
+    dispatch(continueGame(item))
+    console.log(item)
```
[the remark this answers is not in the document]
Comment (KarolCislowski) [reply code/src/components/GameScreen.js:13]: for sure 😺

File: code/src/components/GameScreen.js [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+
+  console.log(actions)
+  return (
+    <section className='game-on'>
+      <div className='head-description'>
+        <h2>{actions.description}</h2> 
+      </div> 
+      <section className='direction-wrapper'>
+            {actions.actions.map(item => (
+        <>
```
Comment (KarolCislowski) [inline code/src/components/GameScreen.js:24]: remember that every element created by iteration needs a key.

You have only one immediate child element (div) so you shouldn't use react fragment here.

File: code/src/components/StartScreen.js [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+
+  const dispatch = useDispatch()
+
+  const onUsernameSet = () => {
+    dispatch(games.actions.setUsername(inputValue))
+    dispatch(generateGame())
+} 
+
+  return (
+    <section className='start-game'>
```
Comment (KarolCislowski) [inline code/src/components/StartScreen.js:17]: It is not nice that you forgot about the form tag. It striped your input from submission on enter, what is the most used sending the form method 😭

**Review by KarolCislowski — COMMENTED** [review]
Great app 🥳

**Pull request closed without merging**